Discussion/Discussion/Presentation/Comments/Responses/ResponsesView.swift

```diff
@@ -59,7 +59,9 @@ public struct ResponsesView: View {
                                     if let comments = viewModel.postComments {
                                         ParentCommentView(
                                             comments: comments,
-                                            isThread: false, onAvatarTap: { username in
```

Comment:
We should use weak self inside closures like [weak self]

Reply:
Thank you for your suggestion. In this particular case, using weak self is not possible because ResponsesView is not a class or a class-bound protocol. The weak keyword can only be applied to class instances or class-bound protocols. Therefore, it is not applicable to structs or other data types like ResponsesView. I hope this clarifies the situation.

Reply:
Yes, you are right i read it as a class.